1-exercises/B-while-loop/exercise.js

```diff
@@ -1,12 +1,22 @@
 /*
     while loops can be useful when you want to execute some code as long as some condition is true.    
 
-    Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-seperated string.
+    Using a while loop, complete the function below so it logs (using console.log) the first n even numbers 
+    as a comma-seperated string.
     The list of numbers should start with 0. n is being passed in as a parameter.
 */
 
 function evenNumbers(n) {
-    // TODO
+  // TODO
+  let i = 0;
+  let j = [];
+  while (i < n && n !== 0) {
+    {
+      j.push(i * 2);
+    }
+    i++;
+  }
+  console.log(j.join());
```

Comment:
```suggestion
  let evens = [];
  while (i < n && n !== 0) {
    {
      evens.push(i * 2);
    }
    i++;
  }
  console.log(evens.join());
```

Might be better to give it a more descriptive variable name, what do you think?

Reply:
You're right, thanks Berkeli